hack/testing/test-upgrade.sh

```diff
@@ -323,6 +338,7 @@ removeFluentdConfigMaps
 removeEsCuratorConfigMaps
 removeAdminCert
 removeCurator
+removeMinMaster
```

Comment:
can you also add this for the second rerun? after that feel free to kick off a test